examples/medplum-demo-bots/src/health-gorilla/README.md

```diff
@@ -48,22 +48,22 @@ You will also need a FHIR `Organization` resource representing each performing l
 }
 ```
 
-If you use multiple performing labs, you will need to create a separate `Organization` resource for each one.
+If you use multiple performing labs, you will need to create a separate `Organization` resource for each one. There are identifiers on many of the resource types which are noted in [sample data](samples.md).
 
 ## Sending Orders
 
 The `send-to-health-gorilla.ts` demonstrates how to convert a `QuestionnaireResponse` into a Health Gorilla order.
 
 The expected answers in the `QuestionnaireResponse` include:
 
-1. `patient` - reference to the `Patient` resource
-2. `practitioner` - reference to the `Practitioner` resource representing the ordering physician
-3. `performer` - referenc to the `Organization` resource representing the performing lab facility
+1. `patient` - reference to the `Patient` resource. For billing to be successful, the `Patient` must have a country
```

Comment:
"For billing to be successful, the `Patient` must have a country"

This feels out of place here.  There are a bunch of random requirements on `Patient`, so it feels weird to call out "country".  Health Gorilla docs include the required fields.  We should either refer to Health Gorilla docs, or have a totally separate section on data requirements.